skll/utils/commandline/compute_eval_from_predictions.py

```diff
@@ -42,67 +50,83 @@ def get_prediction_from_probabilities(
                multiple classes have the same probability, a class is selected randomly.
             2. "expected_value": Calculates an expected value over integer classes and
                rounds to the nearest int.
-
     random_state: int
-        Seed for `RandomState`, used for randomly selecting a class when necessary.
+        Seed for `np.random.RandomState`, used for randomly selecting a class when necessary.
 
     Returns
     -------
-    predicted_class: str or int
-        Predicted class.
+    Union[int, str]
+        The predicted class.
+
+    Raises
+    ------
+    ValueError
+        If ``classes`` does not contain integers and ``prediction_method`` is
+        "expected_value".
```

Comment:
I want to always put raw values in double ticks just so that they show up pre-formatted in the docs, but I know it's a bit pedantic.

```suggestion
       ``"expected_value"``.
```

Reply:
This occurs in other places, but I don't want to leave a bunch of suggestions. I feel like it just shows up better in the documentation if it looks this way, but it's a bit of a PITA.